en/02_Developer_Guides/01_Templates/01_Syntax.md

```diff
@@ -564,39 +570,24 @@ refer directly to properties and methods of the [`Member`](api:SilverStripe\Secu
 
 ### `fortemplate()` and `$Me` {#fortemplate}
 
-If you reference some `ModelData` object directly in a template, the `forTemplate()` method on that object will be called.
-This can be used to provide a default template for an object.
-
-```php
-// app/src/PageType/HomePage.php
-namespace App\PageType;
-
-use Page;
-
-class HomePage extends Page
-{
-    public function forTemplate(): string
-    {
-        // We can also render a template here using $this->renderWith()
-        return 'Page: ' . $this->Title;
-    }
-}
-```
+If you reference some object directly in a template, the `forTemplate()` method on that object will be called. This can be used to provide a default template for an object.
+If the `forTemplate()` method isn't implemented, the system will attempt to cast your object to a string.
+The default implementation of this method on `ModelData` renders the model using templates named after the class or its superclasses.
 
 ```ss
-<%-- calls forTemplate() on the first page in the list and prints Page: Home --%>
+<%-- calls forTemplate() on the first page in the list --%>
 $Pages->First
 ```
 
 You can also use the `$Me` variable, which outputs the current object in scope by calling `forTemplate()` on the object.
 This is especially helpful when you want to directly render items in a list you're looping over.
 
 > [!WARNING]
-> If the object does not have a `forTemplate()` method implemented, this will throw an error.
+> If the object does not have an appropriate template, implement the `forTemplate()` method, or implement [`__toString()`](https://www.php.net/manual/en/language.oop5.magic.php#object.tostring), this will throw an error.
```

Comment:
Objects don't have to be `ModelData` which is why this warning allows for the case where `forTemplate()` isn't implemented.